TravelPro.html

```diff
@@ -0,0 +1,301 @@
+            <li><select>
+                <option value="EN" selected>EN</option>
+                <option value="HIN">HIN</option>
+                <option value="BN">BN</option>
+            </select>    </li>
+        </ul>
+        <!--<section class="options">
+            <a href="#">Destinations</a>
+            <a href="#">Hotels</a>
+            <a href="#">Flights</a>
+            <a href="#">Bookings</a>
+            <a href="#">Login</a>
+            <a class="sign-up">Sign Up</a>
+            <select>
+                <option value="EN" selected>EN</option>
+                <option value="HIN">HIN</option>
+                <option value="BN">BN</option>
+            </select>
+        </section>-->
+    </nav>
+    </header>
+
+
+
+    <section class="banner">
+        <section class="text-part">
+            <p class="sub-line">BEST DESTINATIONS AROUND THE WORLD</p>
+            <p class="banner-text">Travel, enjoy and live a new and full life</p>
+            <p class="sub-text">Built wicket longer admire do barton vanity itself do in it Preferred to sportsmen it engrossed listening. Park gate sell they west hard for the</p>
+            <button>Find out more</button>
+        </section>
+    </section>
+
+
+
+    <section class="services-heading">
+        <section>
+           <p>CATEGORY</p>
+           <p class="offer">We Offer Best Services</p>
+        </section>
+        <img src="./images/plus.png" alt="plus">
+    </section>
+
+
+
+    <section class="services-block">
+        <article class="service">
+            <img src="./images/weather.png" alt="weather">
+            <section class="service-description">
+                <h3>Calculated Weather</h3>
+                <p>Built wicket longer admire do barton vanity itself do in it</p>
+            </section>
+        </article>
+
+        <article class="service">
+            <img src="./images/flight.png" alt="flights" height="80">
+            <section class="service-description">
+                <h3>Best Flights</h3>
+                <p>engrossed listening. Park gate sell they west hard for the</p>
+            </section>
+        </article>
+
+        <article class="service">
+            <img src="./images/event.png" alt="event">
+            <section class="service-description">
+                <h3>Local Events</h3>
+                <p>barton vanity itself do in it Preferred to men it engrossed listening</p>
+            </section>
+        </article>
+
+        <article class="service">
+            <img src="./images/customization.png" alt="custom">
+            <section class="service-description">
+                <h3>Customization</h3>
+                <p>we deliver outsourced aviation services to our customers</p>
+            </section>
+        </article>
+    </section>
+
+
+
+    <section class="destination-heading">
+        <p>Top Selling</p>
+        <p class="heading">Top Destinations</p>
+    </section>
+   
+
+
```

Comment:
Replace the `<section>` tags with `<div>` for city-name and trip-details for more appropriate semantic usage.